tests/test_lab_upload.py

```diff
@@ -0,0 +1,204 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from unittest.mock import Mock, patch
+import pathlib
+import struct
+
+# Third Party
+from click.testing import CliRunner
+from gguf.constants import GGUF_MAGIC
+
+# First Party
+from instructlab import lab
+from instructlab.defaults import DEFAULT_INDENT
+from tests.test_backends import create_safetensors_or_bin_model_files
+
+
+class TestLabUpload:
+    # When using `from X import Y` you need to understand that Y becomes part
+    # of your module, so you should use `my_module.Y`` to patch.
+    # When using `import X`, you should use `X.Y` to patch.
+    # https://docs.python.org/3/library/unittest.mock.html#where-to-patch?
```

Comment:
Was this intended to be pushed?

Reply:
Yup, it's in some of the other test files